io/net/ethtool_test.py.data/README.txt

```diff
@@ -11,7 +11,7 @@ to be updated, so that setting 'up / down <interface>' configures the interface.
 
 Input Needed (in multiplexer file):
 -----------------------------------
-Interfaces     -   Specify the interface for which the test needs to be run.
+device         -   Specify the interface name 'eth1' or mac addr '34:80:0d:9b:dc:30' of device under test
```

Comment:
it is quite confusing why not just go with mac address only

Reply:
the testcase was originally designed for device name, it is good practice to support legacy way so people who already using it should not get disruppted with new way.. mac is optional support and device name is actual way

Reply:
so driving both in single param is confusing though

Reply:
thinking of not changing 2 variables and also no name change.. keep interface as is